MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+include LICENSE
+include README.md
+include CLI.md
+include CONTRIBUTING.md
+include PHILOSOPHY.md
+include pyproject.toml
```

Comment:
Makefile
NOTICE.txt
?

Reply:
Makes sense to add the NOTICE, didn't add the Makefile since it does fit the use case.

Reply:
Just realized that this file isn't doing anything because of the way detection-rules is structured. Based on the [docs](https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html#controlling-files-in-the-distribution) when using include_package_data=True, only files inside the package directory are included. So i'm going to delete it.

Reply:
BTW the alternative is to move the data files all within the detection_rules package.